Makefile

```diff
@@ -6,6 +6,7 @@ test: $(SUBDIRS)
 install-deps:
 	go get -u github.com/golang/lint/golint
 	go get -u github.com/kardianos/govendor
+	go get -u github.com/smartystreets/goconvey
```

Comment:
while there is a number of testing frameworks for golang we should all agree to use only one across all our projects. With golang 1.7 https://blog.golang.org/subtests, native testing framework has been dramatically improved. We should stick to this practice and use it for our dcos-go projects. In past we evaluated testify, ginkgo / gomega and some others. I am, as a contributor would like to see a consistency between modules, and if i need to fix something in your test, i don't want to learn yet another testing framework.

@malnick @alberts and @jdef we had this conversation long time ago, and we decided to use golang testing package for our unittests.